exercises/raindrops/example.go

```diff
@@ -2,8 +2,8 @@ package raindrops
 
 import "strconv"
 
-const testVersion = 3
-
+// Convert converts a number to a string, the contents
+// of which depend on the number's factors.
```

Comment:
As these bonus exercises have no *progression* as such, I wanted to make them as uniform as possible, with stubs similar to `hello-world`.